manager-parent/manager-core/src/main/resources/application.properties

```diff
@@ -54,11 +54,6 @@ event-bridge.kafka.security.protocol=${EVENT_BRIDGE_KAFKA_SECURITY_PROTOCOL:SASL
 event-bridge.kafka.sasl.mechanism=${EVENT_BRIDGE_KAFKA_SASL_MECHANISM:PLAIN}
 event-bridge.resource-prefix=ob-
 
-# WorkManager
```

Comment:
Since the property names and values are the same in `v1` and `v2` why have you removed this from `-core`?

If the names and values were different I'd understand better.